manager/integration/tests/test_settings.py

```diff
@@ -198,3 +198,29 @@ def check_tolerations_set(current_toleration_list, set_tolerations):
                  "operator": t.operator, "effect": t.effect}
 
     return current_tolerations == set_tolerations
+
+
+@pytest.skip(reason="TODO")
+def test_setting_guaranteed_engine_cpu():
+    """
+    Test setting Guaranteed Engine CPU
+
+    1. Change the setting to `0.1`
```

Comment:
Does a change in the "Guaranteed Engine CPU" setting value, result in instance manager pods to be recreated ?

Reply:
Yes.